.github/scripts/build_time_compare.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+file1_url="https://raw.githubusercontent.com/alankritdabral/tailcall/main/benches/iai-callgrind/benchmarks.txt"
```

Comment:
could we specify it as a parameter so we could change it when we will want compare with another branch?